azure/v13.0.0/README.md

```diff
@@ -0,0 +1,186 @@
+#### Other (Cleanup or Flake)
+- Fixes the flooding warning messages about setting volume ownership for configmap/secret volumes ([#92878](https://github.com/kubernetes/kubernetes/pull/92878), [@jvanz](https://github.com/jvanz)) [SIG Instrumentation, Node and Storage]
+- Update CNI plugins to v0.8.7 ([#94367](https://github.com/kubernetes/kubernetes/pull/94367), [@justaugustus](https://github.com/justaugustus)) [SIG Cloud Provider, Network, Node, Release and Testing]
+#### Dependencies
+#### Added
+_Nothing has changed._
+#### Changed
+- github.com/evanphx/json-patch: [162e562 → v4.9.0+incompatible](https://github.com/evanphx/json-patch/compare/162e562...v4.9.0)
+#### Removed
+- github.com/jessevdk/go-flags: [v1.4.0](https://github.com/jessevdk/go-flags/tree/v1.4.0)
+
+
+
+### calico [3.15.3](https://github.com/projectcalico/calico/releases/tag/v3.15.3)
+
+#### Other changes
+ - Add FelixConfiguration parameters to explicitly allow encapsulated packets from workloads. [libcalico-go #1302](https://github.com/projectcalico/libcalico-go/pull/1302) (@doublek)
+ - Respect explicit configuration for drop rules for encapsulated packets originating from workloads. [felix #2487](https://github.com/projectcalico/felix/pull/2487) (@doublek)
+
+
+
+### chart-operator [2.3.5](https://github.com/giantswarm/chart-operator/releases/tag/v2.3.5)
+
+#### Fixed
+- Stop repeating helm upgrade for the failed helm release.
+
+
+
+### app-operator [2.3.3](https://github.com/giantswarm/app-operator/releases/tag/v2.3.3)
+
+#### Added
+- Delete chart-operator helm release and chart CR so it can be re-installed.
+
+
+
+### kube-state-metrics [1.2.0](https://github.com/giantswarm/kube-state-metrics-app/releases/tag/v1.2.0)
+
+#### Added
+- Added monitoring annotations and common labels.
+#### Changed
+- Deploy `kube-state-metrics-app` on installations as part of app collection.
+
+
+
+### cert-exporter [1.3.0](https://github.com/giantswarm/cert-exporter/releases/tag/v1.3.0)
+
+#### Added
+- Add Network Policy.
+#### Changed
+- Remove `hostNetwork` and `hostPID` capabilities.
+
+
+
+### net-exporter [1.9.2](https://github.com/giantswarm/net-exporter/releases/tag/v1.9.2)
+
+#### Changed
+- Updated backward incompatible Kubernetes dependencies to v1.18.5.
+#### Fixed
+- Fixed indentation problem with the daemonset template.
+
+
+
+### node-exporter [1.4.2](https://github.com/giantswarm/node-exporter-app/releases/tag/v1.4.2)
+
+#### Changed
+- Disable `btrfs`,`softnet`,`rapl` and `thermal_zone` to reduce memory usage.
+- Increase memory limit to `75Mi`.
+
+
+
+### external-dns [1.5.0](https://github.com/giantswarm/external-dns-app/releases/tag/v1.5.0)
+
+#### Changed
+- Upgrade upstream external-dns from v0.7.3 to [v0.7.4](https://github.com/kubernetes-sigs/external-dns/releases/tag/v0.7.4).
+
+
+
+### cluster-autoscaler [1.18.2](https://github.com/giantswarm/cluster-autoscaler-app/releases/tag/v1.18.2)
```

Comment:
Should I remove cluster-autoscaler from the readme?

Reply:
yes